company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/row-onboarding.md

```diff
@@ -0,0 +1,19 @@
+## ROW Staff Onboarding
+
+### Full time staff member (Rest of World)
+
+If you are a full time staff member and need to submit an invoice, please follow these instructions so that you will be paid on time. If you're not sure, please check your contract which will denote how you will be paid. You can also reference the [How to get paid](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid) section of the Mattermost handbook for further details.
+
+After you and Mattermost have accepted and signed your offer, you'll go through our full-time staff onboarding process with the [People Team](https://community.mattermost.com/private-core/channels/ask-people-team). As part of your onboarding, please complete the [Banking Form](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid#banking-form) and provide the following tax documentation, as it applies to you:
+
+1. [Form W-9](https://www.irs.gov/pub/irs-pdf/fw9.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw9.pdf) for IRS guidelines\)
+2. [Form W-8BEN](https://www.irs.gov/pub/irs-pdf/fw8ben.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw8ben.pdf) for IRS guidelines\)
+3. [Form W-8BEN-E](https://www.irs.gov/pub/irs-pdf/fw8bene.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw8bene.pdf) for IRS guidelines\)
+
+The Finance Team will get you set up on Accrualify and will send you a Welcome email to the vendor portal. In the vendor portal, you'll be asked for your basic information and banking information. In the portal, you can also upload a copy of your tax document.
+
+If you were part of a paid simulation project, you may have already completed these steps. To conﬁrm you are successfully set up and no further action is required, please email AP@mattermost.com.
+
+### Invoicing and payment
+
+For timely payment, invoices should be sent to AP@mattermost.com or uploaded into the vendor portal. For further information on receiving payment by Mattermost, please refer to the [How to get paid](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid).
```

Comment:
Please delete the section that reads "or uploaded into the vendor portal". This is no longer applicable.